settings.js

```diff
@@ -99,11 +99,11 @@ function init() {
         scratch_layer: true,
         focus: true,
     });
-    defwinprop({
-        wm_class: /gnome-screenshot/i,
-        scratch_layer: true,
-        focus: true,
-    });
+//    defwinprop({
+//        wm_class: /gnome-screenshot/i,
+//        scratch_layer: true,
+//        focus: true,
+//    });
```

Comment:
Can you move this to a separate commit, and maybe a separate PR? I'm a little lukewarm on this change. According to your comment in https://github.com/paperwm/PaperWM/issues/376#issuecomment-962700960 [1], it doesn't seem to trigger reliably. For myself, I cannot reproduce it with gnome-screenshot… but I *have* infrequently experienced gnome-shell crashing and restarting. So, I suspect this bug is not unique to gnome-screenshot, and we'd be better-served trying to find a root cause than just fixing the symptom on one person's computer.

[1]: Side note: GitHub broke this link for me by appending an `=` to the url when I opened it; in order to *actually* jump to the comment, I had to edit the url to remove the `=` and then hit enter.